### PR TITLE
Add env processing and new labels

### PR DESCRIPTION
**Pull request by joshcanhelp**

### CODE REVIEW

File: pr-process/app.js [hunk 1]
```diff
@@ -1,23 +1,41 @@
...
+    { name: 'CH: Added', color: '5319e7', description: 'PR is adding feature or functionality' },
+    { name: 'CH: Breaking Change', color: '5319e7', description: 'PR contains breaking changes without a major version bump' },
+    { name: 'CH: Deprecated', color: '5319e7', description: 'PR is deprecating something in the public API' },
+    { name: 'CH: Fixed', color: '5319e7', description: 'PR is fixing a bug' },
+    { name: 'CH: Changed', color: '5319e7', description: 'PR is changing something' },
+    { name: 'CH: Removed', color: '5319e7', description: 'PR is removing something' },
+    { name: 'CH: Security', color: '5319e7', description: 'PR is security improvement' },
+
+    // PR closing reasons
+    { name: 'closed:stale', color: 'cfd3d7', description: 'Issue or PR has not seen activity recently' },
```
Comment (damieng) [inline pr-process/app.js]: I wonder if we should have a separate file for which labels get created. I think Goalie is a bit specific to auth0 for this non-auth0 repo.

**Review by damieng — CHANGES_REQUESTED** [review]
Love the env var work... Need to remove goalie tho.  I think the CH ones are okay here.

**Comment by damieng** [comment]
Merging for now to get us unblocked but need a more flexible way of customizing the labels.

**Comment by joshcanhelp** [comment]
It's pretty flexible ... just drop an `additional-labels.js` in the same folder (which is ignored):

```js
module.exports = [
    { name: '[Goalie] Review Required', color: 'ccff00', description: 'Goalie triaged, ready for team review' }
];
```

**Pull request merged**